automation/common/helpers.py

```diff
@@ -29,17 +42,30 @@ def run_command(
         process.stdin.write(bytes(stdin, "ascii"))
         process.stdin.close()
 
+    stdout = _handle_command_stdout(process.stdout, log_file_handler, live)
+    stderr = process.stderr.read()
+    exit_status = process.wait()
+
+    return stdout, stderr, exit_status
+
+
+def _handle_command_stdout(
+    stdout_stream: typing.IO[bytes],
+    log_file_handler: typing.IO[str] = None,
+    live: bool = True,
+) -> str:
+    def _write_to_log_file(text: bytes):
+        if log_file_handler:
+            log_file_handler.write(text.decode(sys.stdout.encoding))
+
+    stdout = ""
     if live:
-        for line in iter(process.stdout.readline, b""):
+        for line in iter(stdout_stream.readline, b""):
             stdout += str(line)
             sys.stdout.write(line.decode(sys.stdout.encoding))
```

Comment:
Will this always write to stdout?
The `log-file` flag is for all logs to be written to the file or **both** the file and stdout?

Reply:
I'll clarify in the flag that it's both :)